backend/package.json

```diff
@@ -8,11 +8,12 @@
     "ci": "yarn lint && node bin/test-seed.js && yarn test",
     "dev": "yarn nodemon -L index.js",
     "lint": "yarn eslint src spec",
-    "e2e-seed": "NODE_ENV=test node bin/e2e-seed.js",
+    "e2e-seed": "NODE_ENV=test node bin/e2e-seed-new.js",
     "test": "NODE_ENV=test yarn mocha --reporter dot --check-leaks --timeout 10000 -r spec/specGlobals --recursive spec",
     "tdd": "NODE_ENV=test yarn mocha --reporter dot --check-leaks --timeout 10000 -r spec/specGlobals --recursive spec/unit --watch",
     "seed": "node bin/seed.js && node bin/test-seed.js",
-    "start": "node index.js"
+    "start": "node index.js",
+    "start-test": "NODE_ENV=test node index.js"
```

Comment:
I'm hesitant to add more scripts to the package.json, as it's already a bit out of control. Can we just do `NODE_ENV=test yarn start` anywhere we need this?

Reply:
Personal preference - I see what you mean, but, this one feels like a useful shorthand that would be used relatively commonly, and also serves as documentation for future devs.